pkg/service/catalog/api/types.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+type CatalogService struct {
+	// Target is a reference to a service
+	Target kapi.ObjectReference
+	// Type is a published service's claim type
+	Type string
+	// Description is a user-defined description of the published service
+	Description string
```

Comment:
SGTM. Something like this?

openshift.io/servicecatalog: default
openshift.io/servicecatalog.labels: foo=bar,color=red
openshift.io/servicecatalog.displayname: awesomesvc
openshift.io/servicecatalog.description: this is an awesome service
openshift.io/servicecatalog.annotations: a=1,b=2